framework/src/outputs/ControlOutput.C

```diff
@@ -0,0 +1,151 @@
+
+  // Produce the control information
+  oss << "Controls:\n";
+  for (std::map<MooseSharedPointer<InputParameters>, std::set<MooseObjectName> >::const_iterator iter = objects.begin(); iter != objects.end(); ++iter)
+  {
+    MooseSharedPointer<InputParameters> ptr = iter->first;
+
+    const std::set<std::string> & names = ptr->getControllableParameters();
+
+    if (!names.empty())
+    {
+      oss << "  " <<COLOR_YELLOW << ptr->get<std::string>("_object_name") << COLOR_DEFAULT << '\n';
+
+      // Full names(s)
+      oss << std::setw(ConsoleUtils::console_field_width) << "    Name(s):";
+      for (std::set<MooseObjectName>::const_iterator it = iter->second.begin(); it != iter->second.end(); ++it)
+        oss << (*it) << " ";
+      oss << '\n';
+
+      // Tag(s)
+      const std::vector<std::string> & tags = ptr->get<std::vector<std::string> >("control_tags");
+      if (!tags.empty())
+      {
+        oss << std::setw(ConsoleUtils::console_field_width) <<  "    Tag(s):";
+        for (std::vector<std::string>::const_iterator it = tags.begin(); it != tags.end(); ++it)
+          oss << *it << " ";
+        oss << '\n';
+      }
+
+      oss <<  "    Parameter(s):\n";
+      for (std::set<std::string>::const_iterator it = names.begin(); it != names.end(); ++it)
+        oss << std::setw(ConsoleUtils::console_field_width) << std::string("      ") + *it << ptr->type(*it) << '\n';
+    }
+  }
+
+  _console << oss.str() << std::endl;
+}
+
+
+void
+ControlOutput::outputChangedControls()
+{
+  // Extract InputParameter objects from warehouse
+  InputParameterWarehouse & wh = _app.getInputParameterWarehouse();
+  const std::map<MooseSharedPointer<InputParameters>, std::set<MooseObjectParameterName> > & controls = wh.getControlledParameters();
+
+  // The stream to build
+  std::stringstream oss;
+  oss << std::left;
+
+  // Print header
+  if (!controls.empty())
+    oss << "Active Controls:\n";
+
+  // Loop over the controlled parameters
+  for (std::map<MooseSharedPointer<InputParameters>, std::set<MooseObjectParameterName> >::const_iterator iter = controls.begin(); iter != controls.end(); ++iter)
+  {
+    const MooseSharedPointer<InputParameters> ptr = iter->first;
+    oss << "  " <<COLOR_YELLOW << ptr->get<std::string>("_object_name") << COLOR_DEFAULT << '\n';
+
+    // Tag(s)
+    const std::vector<std::string> & tags = ptr->get<std::vector<std::string> >("control_tags");
+    if (!tags.empty())
+    {
+      oss << std::setw(ConsoleUtils::console_field_width) <<  "    Tag(s):";
+      for (std::vector<std::string>::const_iterator it = tags.begin(); it != tags.end(); ++it)
+        oss << *it << " ";
+      oss << '\n';
+    }
+
+    oss <<  "    Parameter(s):\n";
+    for (std::set<MooseObjectParameterName>::const_iterator it = iter->second.begin(); it != iter->second.end(); ++it)
+      oss << std::setw(ConsoleUtils::console_field_width) << std::string("      ") + it->parameter() << ptr->type(it->parameter()) << '\n';
```

Comment:
I'm a little confused with what you are doing here. You are using `setw()` but then also padding this parameter? Can you just do it all with `setw()`.